packages/web/src/components/ui/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react';
+import styles from './BarChart.module.scss';
+import clsx from 'clsx';
+import { formatNumber } from '../../../utils/helpers';
+import { Icon } from '../Icon/Icon';
+
+type Bar = {
+  fill: string;
+  uses: number;
+  moduleVersion: string;
+  vulnerabilities?: boolean;
+  highlighted?: boolean;
+};
+
+type Props = {
+  bars: Bar[];
+};
+
+const Bar = ({ fill, uses, moduleVersion, vulnerabilities, highlighted }: Bar) => {
+  const memoizedFillHeight = useMemo(() => {
```

Comment:
nit: naming this `fillHeightStyle` would suffice, explicitly stating the memoization doesn't bring any clarity imo